settings.gradle

```diff
@@ -11,10 +11,13 @@ dependencyResolutionManagement {
 
 // sdk modules
 include ":sdk"
-include ":sdk:core", ":sdk:analytics", ":sdk:push-fcm", ":sdk:fixtures"
+include ":sdk:core", ":sdk:analytics", ":sdk:push-fcm", ":sdk:fixtures", ":sdk:messaging"
 
 refreshVersions {
     rejectVersionIf {
         candidate.stabilityLevel.isLessStableThan(current.stabilityLevel)
     }
 }
+include ':sdk'
+include ':sdk:core', ':sdk:analytics', ':sdk:push-fcm', ':sdk:fixtures'
```

Comment:
Are these two lines needed when we have the `include` updated above on line 14 already?